src/monitoring/statistics.h

```diff
@@ -100,7 +100,7 @@ inline void MeasureTime(Statistics* statistics, uint32_t histogram_type,
   }
 }
 
-inline void RecordTick(Statistics* statistics, uint32_t ticker_type,
+inline void RecordTick(std::shared_ptr<Statistics> statistics, uint32_t ticker_type,
```

Comment:
why is this changed to a shared pointer? this was copied straight from rocksdb, right?